src/Microsoft.TemplateEngine.Cli/Commands/GlobalArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal class GlobalArgs
+    {
+        internal bool Quiet { get; private set; }
+
+        internal bool DebugAttach { get; private set; }
+
+        internal bool DebugRebuildCache { get; private set; }
+
+        internal bool DebugVirtualSettings { get; private set; }
+
+        internal bool DebugReinit { get; private set; }
+
+        internal bool DebugShowConfig { get; private set; }
+
+        internal string? DebugSettingsLocation { get; private set; }
```

Comment:
Did we rename some of these? Let's add comments to explain what they mean when the PR is out of draft mode.

And I don't see: --debug-disable-sdk-templates

The ones I have on my notes: --debug-attach, --debug-rebuild-cache, --debug-ephemeral-hive, --debug-reinit, --debug-show-config, --debug-emit-telemetry, --debug-custom-hive, --debug-disable-sdk-templates

Reply:
these are just some examples for now. 
Tricky ones are: `--debug-disable-sdk-templates` and ` --debug-emit-telemetry`: they are used in shim outside of `Cli`. We need to discuss what to do with them